_guides/contextual_bandits.md

```diff
@@ -10,106 +10,117 @@ body_class: tutorial
 resource_icon: /svg/resources/guide.svg
 ---
 
-# Contextual Bandits in VW
+# Contextual Bandits and Vowpal Wabbit
 
-The goal of this tutorial is to have you walk away with an understanding of Contextual Bandits, when CB can be used, how to run CB algorithms in Vowpal Wabbit (VW), and hopefully make you feel empowered and excited to use it on your own. This tutorial focuses on Python but VW is also supported in C++ and C#.
+This guide describes how to use Contextual Bandit (CB) algorithms in Vowpal Wabbit (VW). It features a Python tutorial and an overview of CB algorithms—including when to use them, how to work with with different CB approaches, how to format test data, and understand the results. 
+
+## Getting started with Vowpal Wabbit
+
+To install VW—and for more information on building VW from source or using a package manager—see [Getting started](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Getting-started) guide. 
+
+>**Note:** The CB tutorial uses [Vowpal Wabbit Python package](https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/python). Additional binary packages are available for select platforms. See [Getting started](https://cheng-tan.github.io/vowpalwabbit.github.io/) module on the homepage for more information.
 
 ## What is a Contextual Bandit?
 
-Consider an application that interacts with its environment, such as a news website with users or a cloud controller with machines. Let's call this application _APP_. This application repeatedly goes through the following:
-1. Some context x arrives and is observed by _APP_
-2. _APP_ chooses an action _a_ from a set of actions _A_ i.e. a ∈ A to take (A may depend on x)
-3. Some reward r for the chosen a is observed by _APP_
+Contextual Bandits are algorithms that use additional information (or context) to make better decisions when choosing actions. Bandit algorithms aid decision-making by making smarter choices in dynamic environments where options change rapidly, and the set of available actions is limited. 
```

Comment:
Contextual Bandit by itself isn't an algorithm. I think we should not talk about CB algorithms till we make clear what CB is. I think we should stick to starting with the APP example.